core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerBuilder.java

```diff
@@ -71,8 +71,9 @@
     /**
      * Creates a new {@link CircuitBreakerBuilder} with the specified name.
      *
-     * @param name The name of the circuit breaker.
+     * @deprecated Use {@link CircuitBreaker#builder()} instead.
```

Comment:
Could you remove `instead`? we don't use that. If you find it another class, please remove it as well.

Reply:
Ok, I ended sentence with "instead. ", since I saw several classes end sentences like that. 
For example, RequestContextExportingAppender, RequestContext  etc.. 

/**  

\* Registers {@code callback} to be run when re-entering this {@link RequestContext}, usually when using
     * the {@link #makeContextAware} family of methods. Any thread-local state associated with this context
     * should be restored by this callback.
     *
     * @deprecated Use {@link #onEnter(Consumer)} instead.
     */


Do you think we could remove the 'instead."s or they use "instead." properly?

Reply:
As I said before, please remove all of them. 🙇

Reply:
I got it.  let me make issue for that!

Reply:
Some time ago, we put `instead` at the end of the sentence but we started to remove them based on the feed back from @anuraaga :-)

Reply:
Oh, I see. It would be great to me to take that job after I wrap up this thing.